devmapper/devmapper.go

```diff
@@ -501,7 +521,7 @@ func (d *Driver) CreateVolume(req Request) error {
 	}
 	if backupURL == "" {
 		// format the device
-		if _, err := util.Execute("mkfs", []string{"-t", "ext4", dev}); err != nil {
+		if err := d.createFilesystem(dev); err != nil {
```

Comment:
I meant, we're using the same command to format the disk, the only change part is "ext4" or "xfs" or others. 

So you don't need to introduce another function for it.

Or at least, combine the "ext4" and "xfs" option in `createFilesystem` in some way that you don't need to refer to `mkfs` twice. It will make it easy for more filesystems in the future. The new option will only need to be checked for `fsSupported`, later it should be automatically picked up without change a line of code in format, as long as it's using `mkfs`.